inlanecruising_plugin/include/inlanecruising_plugin/smoothing/BSpline.h

```diff
@@ -18,25 +18,27 @@
 
 #include <vector>
 #include <carma_wm/Geometry.h>
-#include <inlanecruising_plugin/third_party_library/spline.h>
 #include <inlanecruising_plugin/smoothing/SplineI.h>
+#include <unsupported/Eigen/Splines>
```

Comment:
I'm curious to know more what "unsupported" here means, do we have a feel for what the implications of using this are? Does it run the risk of being removed or broken in later versions?